lahja/base.py

```diff
@@ -655,6 +655,19 @@ def get_connected_endpoints_and_subscriptions(
             for remote in self._connections
         )
 
+    def maybe_raise_no_subscribers_exception(
```

Comment:
Seems like this should be namespaced as a private API.

Reply:
It can not be *private*  because because we want to call it from inherited classes. And if we mark it private it will fall out of the docs (rightfully so). In Java or .NET terms, this would be *protected*  or *internal* if we want to express it in Solidity terms. I've had this a few times before that I really wished we had a convention on how to prefix *protected* members. Anything that isn't an underscore to keep it in the docs. But a prefix like `protected` or `internal` may make an odd combination with the actual function name and also eats quite a bit of horizontal space...